chapter06/kds/ex3.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Printing ENV hash:\n";
+
+printf "%-" . column_size() . "s => %s\n", $_, $ENV{$_} foreach ( sort keys %ENV );
```

Comment:
Yeah I really like the idea of it, I was wondering if you could do something with state to hold it in the sub but it might not gain much